src/components/Notification.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { ErrorType } from '../types/ErrorType';
+
+type Props = {
+  errorType: ErrorType,
+};
+
+export const Notification: React.FC<Props> = ({ errorType }) => {
+  const [isButtonHidden, setIsButtonHidden] = useState(false);
+  const handleNotification = () => {
+    setIsButtonHidden(true);
```

Comment:
Consider renaming 'handleNotification' to 'hideNotification' for better clarity.